server/controllers/votes/VoteController.js

```diff
@@ -0,0 +1,89 @@
+import models from '../../models';
+import {
+  ARTICLE_LIKE_MSG,
+  ARTICLE_DISLIKE_MSG,
+  ARTICLE_UNLIKE_MSG
```

Comment:
what happen to my feedback about this?

Reply:
It has been implemented. Sorry, it took longer than expected.